protocol/0080-SPOT-product_builtin_spot.md

```diff
@@ -121,6 +121,11 @@ Price-monitoring auctions are still required and should be implemented following
 1. If a "sell" order does not trade immediately (or only trades in part), an amount of the base_asset should be transferred to a holding_account to cover the remaining size of the order for the base_asset.(<a name="0080-SPOT-009" href="#0080-SPOT-009">0080-SPOT-009</a>)
 1. If a "sell" order incurs fees through trading, the required amount of the quote_asset to cover the fees will be deducted from the total quote_asset resulting from the sale of the base_asset.(<a name="0080-SPOT-010" href="#0080-SPOT-010">0080-SPOT-010</a>)
 1. For a "buy" order to be considered valid during continuous trading, the party must have a sufficient amount of the `quote_asset` in the `general_account` to cover the value of the trade as well as any possible fees incurred as a result of the order trading immediately (the aggressor).(<a name="0080-SPOT-012" href="#0080-SPOT-012">0080-SPOT-012</a>)
+1. For a "buy" market order to be considered valid during continuous trading, the party must have a sufficient amount of the `quote_asset` in the `general_account` to cover the value of the trade as well as any possible fees incurred as a result of the order trading immediately (the aggressor).(<a name="0080-SPOT-024" href="#0080-SPOT-024">0080-SPOT-024</a>)
+1. For a "sell" market order to be considered valid during continuous trading, the party must have a sufficient amount of the `base_asset` in the `general_account` to cover the value of the trade as well as any possible fees incurred as a result of the order trading immediately (the aggressor).(<a name="0080-SPOT-025" href="#0080-SPOT-025">0080-SPOT-025</a>)
+1. amending order should be rejected when an order is amended such that would trade immediately and the party can't afford none/some of the trades(<a name="0080-SPOT-026" href="#0080-SPOT-026">0080-SPOT-026</a>)
+1. order should be rejected when submit a limit order, partly matched, party can't afford the trades.(<a name="0080-SPOT-027" href="#0080-SPOT-027">0080-SPOT-027</a>)
```

Comment:
Maybe as a separate AC cover the case where a party can afford to cover the partial trade **but** not what needs to be transferred to the holding account after to cover the remaining size.